Lib/test/_test_multiprocessing.py

```diff
@@ -752,6 +752,16 @@ def test_timeout(self):
         # Windows (usually 15.6 ms)
         self.assertGreaterEqual(delta, 0.170)
 
+    def test_queue_feder_donot_stop_onexc(self):
+        class NotSerializable(object):
+            def __reduce__(self):
+                raise AttributeError
+        with test.support.captured_stderr():
+            q = multiprocessing.Queue()
```

Comment:
This test is only for multiprocessing.Queue, manager and threading queues implementations do not start feeder.